README.md

```diff
@@ -52,3 +52,150 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Steps
+
+**Module 1**
```

Comment:
You shouldn't have steps for each module. The README should include info like how to run the app and tests, for the current version of the codebase.

i.e. keep it up to date, don't divide it into sections some of which are incorrect because of later sections. That's not useful for a new developer joining the team trying to run the app.

If you want to remind yourself what you did in earlier modules, that's fine, but not the purpose of a README